### PR TITLE
feat: Enable Modals in Header/Footer

### PR DESCRIPTION
**Pull request by andreituicu**
Test URLs:
- Before: https://main--helix-block-collection--adobe.hlx.page/block-collection/search
- After: https://modal-header--helix-block-collection--adobe.hlx.page/block-collection/search

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -93,6 +92,7 @@ async function loadEager(doc) {
   if (getMetadata('breadcrumbs').toLowerCase() === 'true') {
     document.body.classList.add('breadcrumbs-enabled');
   }
+  autolinkModals(doc);
```
Comment (shsteimer) [inline scripts/scripts.js:95]: very minor/nitpicky, but I'd probably do this after LCP, and maybe even in loadLazy.

File: blocks/header/header.css [hunk 2]
```diff
@@ -279,6 +279,13 @@ header nav#nav .nav-tools {
   grid-area: tools;
 }
 
+header nav#nav .nav-tools .button {
+  background: none;
+  border: none;
+  padding: 0;
```
Comment (synox) [inline blocks/header/header.css:285]: I would set the padding, so that it is easier to click
```suggestion
  padding: 10px;
```

**Review by synox — APPROVED** [review]
looks great, I would just increase the padding

File: scripts/scripts.js (hunk 1 above)
Comment (andreituicu) [reply scripts/scripts.js]: Done in https://github.com/adobe/helix-block-collection/pull/37/commits/70a0d5fa0fdf86c06623297ef2542026eb4cb673 👍

File: blocks/header/header.css (hunk 2 above)
Comment (andreituicu) [reply blocks/header/header.css]: Done in https://github.com/adobe/helix-block-collection/pull/37/commits/70a0d5fa0fdf86c06623297ef2542026eb4cb673 👍

**Pull request merged**